### ISSUE TITLE
Use proper aria label

### ISSUE THREAD

**Issue by ewmcelvaney** [CodefortheCarolinas/tax-benefit-website#49]
The section with the map has an aria-label that would be better suited as alt text. There should be a new aria label put in once the current aria label has been converted to alt text (I will edit this comment when I figure out what the proper aria label is because the current label was marked as invalid by my accessibility checker). 

This is the area where the issue was found:
<div class="category-legend horizontal" role="graphics-legend" aria-label="Category legend with 2 items. The items are City plan, and Statewide plan" style="padding-top: 0px; padding-bottom: 10px; text-align: left; width: 306px; height: auto;">
